blocks/cards/cards.js

```diff
@@ -10,7 +10,29 @@ export default function decorate(block) {
     while (row.firstElementChild) li.append(row.firstElementChild);
     [...li.children].forEach((div) => {
       if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+      else { div.className = 'cards-card-body'; }
+
+      /* cards simple */
+      if (block.classList.contains('simple')) {
+        const anchorEl = div.querySelector('a');
+        if (anchorEl) {
+          anchorEl.textContent = '';
+          div.append(anchorEl);
+          div.querySelector('.button-container').remove();
+        }
+        const cardBodyContent = document.createElement('div');
```

Comment:
Can you use dom-helpers "div"? you'll be able to add the className in the same line (you'll have to either alias the import or rename the div variable though)

Reply:
Updated using dom-helper.